install.sh

```diff
@@ -94,133 +103,128 @@ esac
+  echo '  "architecture": "'"${ARCH}"'",' >> device.json
   echo '  "installed_packages": [' >> device.json
   echo '  ]' >> device.json
   echo '}' >> device.json
 fi
 
 # extract, install and register packages
-for i in `seq 0 $((${#urls[@]} - 1))`; do
-  url=${urls[$i]}
-  sha256=${sha256s[$i]}
-  tarfile=`basename $url`
-  name=${tarfile%%-*}   # extract string before first '-'
-  rest=${tarfile#*-}    # extract string after first '-'
-  version=`echo $rest | sed -e 's/-chromeos.*$//'`
+for i in $(seq 0 $((${#urls[@]} - 1))); do
+  url="${urls["${i}"]}"
+  sha256="${sha256s["${i}"]}"
+  tarfile="$(basename ${url})"
+  name="${tarfile%%-*}"   # extract string before first '-'
+  rest="${tarfile#*-}"    # extract string after first '-'
+  version="$(echo ${rest} | sed -e 's/-chromeos.*$//')"
                         # extract string between first '-' and "-chromeos"
 
-  download_check $name $url $tarfile $sha256
-  extract_install $name $tarfile
-  update_device_json $name $version
+  download_check "${name}" "${url}" "${tarfile}" "${sha256}"
+  extract_install "${name}" "${tarfile}"
+  update_device_json "${name}" "${version}"
 done
 
-# download, prepare and install chromebrew
-cd $CREW_LIB_PATH
-rm -rf crew lib packages
-curl -# -o crew $URL/crew
-chmod +x crew
-rm -f $CREW_PREFIX/bin/crew
-ln -s `pwd`/crew $CREW_PREFIX/bin
-#install crew library
-mkdir -p $CREW_LIB_PATH/lib
-cd $CREW_LIB_PATH/lib
-curl -# -o package.rb $URL/lib/package.rb
-curl -# -o package_helpers.rb $URL/lib/package_helpers.rb
-
-# Making GCC act like CC (For some npm packages out there, only required for gcc)
-#rm -f $CREW_PREFIX/bin/cc
-#ln -s $CREW_PREFIX/bin/gcc $CREW_PREFIX/bin/cc
-
-# package gcc7 has already made symbolic links for cc and gcc, no action required here
+# create symlink to 'crew' in ${CREW_PREFIX}/bin/
+rm -f "${CREW_PREFIX}/bin/crew"
+ln -s "../lib/crew/crew" "${CREW_PREFIX}/bin/"
 
 # prepare sparse checkout .rb packages directory and do it
-cd $CREW_LIB_PATH
+cd "${CREW_LIB_PATH}"
 rm -rf .git
 git init
-git remote add -f origin https://github.com/$OWNER/$REPO.git
+git remote add -f origin "https://github.com/${OWNER}/${REPO}.git"
 git config core.sparsecheckout true
 echo packages >> .git/info/sparse-checkout
 echo lib >> .git/info/sparse-checkout
 echo crew >> .git/info/sparse-checkout
-git fetch origin master
-git reset --hard origin/master
+git fetch origin "${BRANCH}"
+git reset --hard origin/"${BRANCH}"
 
 # install a base set of essential packages
 yes | crew install buildessential less most
 
+echo
+echo "If you have installed Chromebrew in a directory other than '/usr/local',"
+echo "please run these commands:"
+echo "echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc && . ~/.bashrc"
+echo "echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc && . ~/.bashrc"
+echo "echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc && . ~/.bashrc"
```

Comment:
How about this?
```
if "${CREW_PREFIX}" != "/usr/local" then
  echo "echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc"
  echo "echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc"
  echo "echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc"
  source ~/.bashrc
fi
```
No need to print unless they have changed `${CREW_PREFIX}` and no need to source every line until the end.

Reply:
I don't think you want to `source ~/.bashrc` directly from the script :). I'll push this soon.

Reply:
Yes, edited.